src/main/kotlin/uk/co/craigbass/pratura/boundary/AddShippingAddress.kt

```diff
@@ -0,0 +1,17 @@
+package uk.co.craigbass.pratura.boundary
+
+import com.madetech.clean.usecase.SynchronousUseCase
+import uk.co.craigbass.pratura.boundary.AddShippingAddress.Request
+
+interface AddShippingAddress : SynchronousUseCase<Request, Unit> {
+  data class Request(
+    val name: String,
+    val companyName: String?,
+    val addressLine1: String,
+    val addressLine2: String?,
```

Comment:
I'm wondering where this should be `addressLines: List<String>` so addresses can have arbitrary amounts of lines... but then we'd need to enforce validation rules to occur e.g. you must have at least one line